packages/dds/tree/.mochaRunCustomData.cjs

```diff
@@ -0,0 +1,36 @@
+/*!
+ * Copyright (c) Microsoft Corporation and contributors. All rights reserved.
+ * Licensed under the MIT License.
+ */
+
+/**
+ * Mocha configuration file to run memory-profiling tests
+ */
+"use strict";
+
+const getFluidTestMochaConfig = require("@fluid-internal/mocha-test-setup/mocharc-common");
+
+const packageDir = __dirname;
+const baseConfig = getFluidTestMochaConfig(packageDir);
+
+const nodeOptions =
+	baseConfig["node-option"] !== undefined
+		? Array.isArray(baseConfig["node-option"])
+			? baseConfig["node-option"]
+			: [baseConfig["node-option"]] // If string, wrap as array to use spread operator
+		: []; // If undefined, use an empty array
+
+nodeOptions.push("expose-gc", "gc-global", "unhandled-rejections=strict");
+
+module.exports = {
+	...baseConfig,
+	"exit": true,
+	"fgrep": ["@Benchmark", "@CustomBenchmark"],
+	"node-option": nodeOptions, // without leading "--"
+	"recursive": true,
+	"reporter": "@fluid-tools/benchmark/dist/MochaReporter.js",
+	"reporterOptions": ["reportDir=.memoryTestsOutput/"],
+	"require": ["node_modules/@fluid-internal/mocha-test-setup"],
```

Comment:
I think this one is unnecessary when we do `const getFluidTestMochaConfig = require("@fluid-internal/mocha-test-setup/mocharc-common");`. Other custom mocha configs (for memory benchmarks, I think) don't make that call so they need to explicitly require this, but in this case we should be able to remove it.
```suggestion
```